README.md

```diff
@@ -0,0 +1,33 @@
+# Dwellingly Frontend
+
+This serves as the Frontend of the Full Stack Application for [JOIN Organization's](https://joinpdx.org/) Dwellingly site being created per the FIGMA designs. This site will eventually aid in supporting both tenants
```

Comment:
Should we put a link to the Figma PDF here?

Reply:
Absolutely! I had it originally and wasn't sure if I should.